### PR TITLE
Readme revamp

### PR DESCRIPTION
**Pull request by akulkarni**
Re-organized the README:

* Added introduction from the design doc

* Added short intro section on TimescaleDB

* Re-organized the latter sections for readability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,111 @@
...
+Timescale-Prometheus Connector, and Grafana
 into your Kubernetes cluster:
 ```
 helm repo add timescale https://charts.timescale.com/
 helm repo update
 helm install --devel <release_name> timescale/timescale-observability
 ```
 
-# Configuring Helm Chart
+## Getting started (non-Helm)
```
Comment (cevian) [inline README.md:61]: I think all the non-helm stuff belongs in advanced. It's also a bit weird to discuss configuring non-helm installation before discussing how to install  them.

Comment (cevian) [reply README.md:61]: I think the old organization is more organic

File: README.md [hunk 2]
```diff
@@ -58,92 +122,85 @@ Note: The default applies to all metrics that do not have override,
...
 
 We describe how to use our pre-defined views and functions to work with the prometheus data in [the SQL schema doc](docs/sql_schema.md).
 
 A Reference for our SQL API is [available here](docs/sql_api.md).
 
-# Advanced
+## Advanced
 
-## Helm (sub)chart for Timescale-Prometheus Connector only
+### Configuring the Helm Chart
```
Comment (cevian) [inline README.md:133]: Moving this here probably makes sense.

File: README.md [hunk 3]
```diff
@@ -1,47 +1,111 @@
...
+
+This platform is **horizontally-scalable**, **highly-compressed**, and
+**operationally mature**, allowing a user
+to ask any question, create any dashboard, and achieve greater
+visibility into their metrics.
+
+For a detailed description of this architecture, please see [our design
+doc][design-doc].
+
+### About TimescaleDB
```
Comment (cevian) [inline README.md:21]: I like this section

File: README.md (hunk 1 above)
Comment (cevian) [reply README.md:61]: If you want a section here for non-helm install I would just describe the steps for non-helm installs and link to relevant advanced sections. Something like:
1) Install and run TimescaleDB (link to tsdb github repo)
2) Install (link), configure (link) and run the connector.
3) Run the cron job for the connector (link) 
4) Configure the Prometheus server to write to the connector (link).
5) Run Prometheus

Comment (akulkarni) [reply README.md]: Updated by just moving non-Helm section back to Advanced

File: README.md [hunk 4]
```diff
@@ -128,22 +167,41 @@ remote_write:
...
 
-The Timescale-Prometheus Connector Binary is configured either through CLI flags or environment variables.
-The list of all available flags is displayed on the help `timescale-prometheus -h` command. All
+The Timescale-Prometheus Connector Binary is configured either through
+CLI flags or environment variables. The list of all available flags is
+displayed on the help `timescale-prometheus -h` command. All
 environment variables are prefixed with `TS_PROM`.
 
-## Building
+### Binaries
```
Comment (cevian) [inline README.md:181]: I think this and the following 2 sections belong under "Non-Helm installation methods" These do not matter for helm. Not sure why they were moved.

Comment (akulkarni) [reply README.md:181]: Is it just the "Binaries" and "Docker" sections? Because "Building from source" was a higher-level section before.

Comment (akulkarni) [reply README.md:181]: Note after chatting with Mat: Keep "Building from source" separate. Put all 3 sections under non-helm.

**Pull request merged**